modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -996,6 +996,30 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
                              CV_OUT std::vector<int>& indices,
                              const float eta = 1.f, const int top_k = 0);
 
+     class CV_EXPORTS_W_SIMPLE Model : public Net
+     {
+     public:
+         Model(const std::string& model, const std::string& config = "", const Size& size = {-1, -1},
+               const Scalar& mean = {}, float scale = 1.0, bool swapRB = true, bool crop = false);
+
+         Model(const Net& network, const Size& size = {-1, -1},
+               const Scalar& mean = {}, float scale = 1.0, bool swapRB = true, bool crop = false);
+
+         Model& setInputSize(const Size& size);
+         Model& setInputMean(const Scalar& mean);
+         Model& setInputScale(float scale);
+         Model& setInputCrop(bool crop);
+         Model& setInputSwapRB(bool swapRB);
+         std::pair<int, float> classify(InputArray frame);
+         void detect(InputArray frame, CV_OUT std::vector<int>& classIds,
+                     CV_OUT std::vector<float>& confidences, CV_OUT std::vector<Rect2d>& boxes,
+                     float confThreshold, float nmsThreshold = 0.4, bool absoluteCoords = true);
```

Comment:
As one of the goals for this PR is to decrease number of actions made by user to use DL networks. Currently this method returns detections in relative coordinates, right? That means in the most of scenarios users should multiple them by frame's sizes. So in my point of view we need to return `cv::Rect` with pixel coords.

Reply:
if  `absoluteCoords = true` - absolute coordinates (default), else relative coordinates